Sources/XcodeGenKit/CarthageDependencyResolver.swift

```diff
@@ -159,6 +159,8 @@ extension Platform {
             return "watchOS"
         case .macOS:
             return "Mac"
+        case .visionOS:
+            fatalError("Not supported.")
```

Comment:
Carthage seems not to be maintained. so it will be unlikely to support visionOS.
  
  We can just return a suitable value like `visionOS` or raise an error when a visionOS target attempts to use Carthage dependencies.
  
  ```suggestion
              fatalError("Carthage doesn't support visionOS.")
  ```

Reply:
I decided to use a dummy value because some test codes would be complicated. Fixed: [e66fb5b](https://github.com/yonaskolb/XcodeGen/pull/1379/commits/e66fb5be7672b2c97c83d0956ecf5e522cab78a2)